src/bindings/js/node/src/node_wrap.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "node/include/node_wrap.hpp"
+
+#include "node/include/errors.hpp"
+#include "node/include/type_validation.hpp"
+
+Napi::FunctionReference NodeWrap::constructor;
+
+Napi::Function NodeWrap::get_class(Napi::Env env) {
+    return DefineClass(env, "NodeWrap", {InstanceMethod("getName", &NodeWrap::get_name)});
+}
+
+NodeWrap::NodeWrap(const Napi::CallbackInfo& info) : Napi::ObjectWrap<NodeWrap>(info) {
+    Napi::Env env = info.Env();
+    std::vector<std::string> allowed_signatures = {"NodeWrap(Napi::External<ov::Node>)", "NodeWrap()"};
```

Comment:
Allowed signatures it's array that fill fulfill by validate method. It needs to show allowed signatures in the case of miss much.  There is no need to add anything to it here.
```suggestion
    std::vector<std::string> allowed_signatures;
```